pkg/distro/defs/rhel-8/distro.yaml

```diff
@@ -524,6 +524,17 @@
             - "insights-client"
             - "subscription-manager-cockpit"
     
+image_config:
+  timezone: "America/New_York"
+  locale:   "en_US.UTF-8"
+  sysconfig:
+    networking: true
+    no_zero_conf: true
+  default_kernel: "kernel"
+  update_default_kernel: true
+  kernel_options_bootloader: true
+  install_weak_deps: true
+  default_oscap_datastream: "/usr/share/xml/scap/ssg/content/ssg-rhel8-ds.xml"
```

Comment:
Currently not all items are consistently ordered, e.g. the `default_oscap_datastream`.

While the order may not matter strictly speaking, it might help with readability.